dev/TeachingTip/TeachingTip.cpp

```diff
@@ -25,6 +26,12 @@ winrt::AutomationPeer TeachingTip::OnCreateAutomationPeer()
     return winrt::make<TeachingTipAutomationPeer>(*this);
 }
 
+void TeachingTip::ClosePopupOnUnloadEvent(winrt::IInspectable const& /* sender */, winrt::RoutedEventArgs const& /* e */)
```

Comment:
>/* sender */ [](start = 69, length = 12)

The other handlers with unnamed parameters in this file don't include comments with the names.  The c++ guidelines don't explicitly indicate if these should or shouldn't be there but for consistency I would delete the comments.

[https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-unused](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-unused)    F.9

In general if you should only include a commented name if you anticipate a need for the parameter in the future, which I do not think you will.

Reply:
I removed the comments with the latest commit.